renderers/svglayers.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/renderers/svglayers.schema.json",
+    "title": "Layered SVG Renderer",
+    "type": "object",
+    "description": "Data for verlay visualization of data from an input graphic, possibly containing multiple layers. ca.mcgill.a11y.image.renderer.SVGLayers.",
+    "definitions": {
+        "svgLayer": {
+            "description": "A single SVG graphic encoded as a data URL.",
+            "type": "string",
+            "pattern": "^data:image\/svg\\+xml(;base64)?,[^ :/?#[\\]@!$&'()*+,;=]+$"
+        }
+    },
+    "properties": {
+        "layers": {
+            "description": "The layer(s) representing data. It should be possible to overlay each layer on the original graphic to meaningfully convey the desired information.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "svg": { "$ref": "#/definitions/svgLayer" },
+                    "label": {
+                        "type": "string",
+                        "maxLength": 300,
```

Comment:
Assuming this is the string that will be displayed in the UI, in a dcropdown, 300 is probably too many?

Reply:
Probably too many, but I'm more worried about reasonable labels being rejected for going slightly over a limit than them ending up with someone trying to put in a novella! Do you have a suggestion to set it to?

Reply:
Hm. This is primarily for debugging, so it can be longer than something I'd consider end-user UI. Looking at current window, even if dropdown is all the way across the screen, I'd say 200 is more than enough, and 100 would enforce more discipline in naming. 150?